src/db-fixture-pg.ts

```diff
@@ -31,108 +21,100 @@ export class PGFixture
         let tool1_callback = (tool_id, tool_name, area_id, member_id) => {
             this.tool1ID = tool_id;
             this.tool1Name = tool_name;
-            this.addTool( db, area_id, member_id,
-                (tool2_id, tool2_name) => tool2_callback(
-                    tool2_id, tool2_name, area_id, member_id )
+            this.addTool(db, area_id, member_id, (tool2_id, tool2_name) =>
+                tool2_callback(tool2_id, tool2_name, area_id, member_id),
```

Comment:
Here, I don't like the original or the new version. Parameters should break out line by line.